README.md

```diff
@@ -568,8 +587,17 @@ export default async function BlogIndex() {
 
   const posts: BlogPage[] = data.items;
 
-  // The rest stays the same
-  // ...
+  return (
+    <main>
+      <div className="mb-8">
+        <h1 className="text-4xl font-bold mb-2">{index.title}</h1>
+        <div dangerouslySetInnerHTML={{ __html: index.intro }}></div>
```

Comment:
MhmhmmHmhmhmhmH, I’d probs recommend skipping this, as this won’t work for anything but the most basic markup – and making it work is way more work, I believe. Unless Wagtail has changed to now output "rendered" HTML by default in the API.

Reply:
We can have a discussion on the fly about it if it's still bugging you when we get to that part of the workshop. I'm sure people would like to hear why you're not keen on it.